v0/destinations/zendesk/transform.js

```diff
@@ -45,6 +47,100 @@ function responseBuilder(message, headers, payload, endpoint) {
   return response;
 }
 
+/**
+ * Returns the payload for updating primary email of users.
+ * @param {*} userIdentityId -> userIdentity Id
+ * @param {*} userId -> userId of users
+ * @param {*} headers -> Authorizations for API's call
+ * @returns
+ */
+const responseBuilderToSetPrimaryAccount = (
```

Comment:
Can't we use the common response builder instead of creating another here?

Reply:
for both the builder function the common part is headers, other than that all were different. So, I've separated it for getting its clear use.